doc/releasenotes/12_0_4_release_notes.md

```diff
@@ -0,0 +1,14 @@
+# Release of Vitess v12.0.4
+## Changelog
```

Comment:
Can you list the bugs/errors people might encounter that are fixed by this release?
The PRs may have links to issue that will help with this.